app/javascript/crayons/formElements/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,43 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+export const RadioButton = ({
```

Comment:
QQ: Do you think it would make sense to call it simply `Radio` instead of `RadioButton`? I was never sold on attaching _button_ keyword to radios..

Reply:
I still think it makes sense to call it `RadioButton` because that's what it actually is. It's not a radio (not being sarcastic 😉). Having said that, I would still understand what it was if it was called `Radio` though. I'll pop a little poll on Slack.

Reply:
The internal poll stuck with `<RadioButton />`